ci/playbooks/collect_logs.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- hosts: controller
+  gather_facts: false
+  tasks:
+    - name: Create log dir
+      file:
+        path: "{{ ansible_user_dir }}/zuul-output/logs"
+        state: directory
+        mode: 0755
+
+    - name: Collect pod logs
+      ansible.builtin.shell: |
+        source ~/.bashrc
+        pushd {{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls
+        cp -R out {{ ansible_user_dir }}/zuul-output/logs
```

Comment:
we will want to be more selective then this

the out dir has all the git repos for all the operators that were deployed

we only want to copy the generated files.

Reply:
Yes, I have updated the pull request to copy only generated files.